ocs_ci/helpers/dr_helpers.py

```
@@ -1864,10 +1806,19 @@ def get_managed_cluster_node_ips():
     ]
     for cluster in cluster_data:
         config.switch_ctx(cluster[1])
-        logger.info(
-            f"Reading fencing CIDRs from DRClusterConfig on managed cluster {cluster[0]}"
-        )
-        cluster.append(get_fencing_cidrs_from_drclusterconfig(cluster[0]))
+        logger.info(f"Getting worker node IPs on managed cluster: {cluster[0]}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)